components/NavBar/index.js

```diff
@@ -33,20 +32,20 @@ const NavBar = () => {
         })
         .then((responseJson) => {
           if (responseJson.incompleteUserDetails) {
-            return window.location.replace(
-              'https://my.realdevsquad.com/signup'
-            );
+            window.open('https://my.realdevsquad.com/signup', '_blank');
           }
           setIsLoggedIn(true);
           setUserData({
             userName: responseJson.username,
             firstName: responseJson.first_name,
+            profilePicture: responseJson.picture
+              ? responseJson.picture.url
```

Comment:
I think we can use optional chaining here. What do you think ??

Reply:
Yes, combining Optional chaining and Nullish coalescing operator will work
```js
responseJson.picture?.url ?? DEFAULT_AVATAR
```
I will make the changes

Reply:
Yes, that would be great.

Reply:
Done